.gitignore

```diff
@@ -20,4 +20,8 @@
 # macOS
 *.DS_store
 
+# Editors
+.vscode
+
 .bin
+/kustomize/kustomize
```

Comment:
why add this?

Reply:
Because `go build` inside `kustomize/` produces a `kustomize` binary that should be ignored by git.

Reply:
Any particular reason you're using `go build` instead of the [make target](https://github.com/kubernetes-sigs/kustomize/blob/master/Makefile#L85)? NB this repo can produce a variety of other binaries if built manually, notably from the `cmd/` subdirectories.